src/host/ut_host/VtRendererTests.cpp

```diff
@@ -714,23 +827,21 @@ void VtRendererTest::Xterm256TestExtendedAttributes()
         onSequences.push_back("\x1b[2m");
         offSequences.push_back("\x1b[22m");
     }
+
+    // underlined and doublyUnderlined are mutually exclusive
     if (underlined)
     {
-        desiredAttrs.SetUnderlined(true);

```

Comment:
Are we losing coverage of normal single underlines here? Or is it covered elsewhere?

Reply:
It looks like it is covered below in the Reset test, so that's good. I don't know how much I care about fixing this one. WDYT?

Reply:
If there's nothing special about singly underlined that needs to be tested for that particular test, I just swapped the singly underlined with another style, so it's clear that other styles works as intended aswell. I think this is also one of those cases.

Reply:
Fair enough!